mvvmfx/src/test/java/de/saxsys/mvvmfx/utils/notifications/DefaultNotificationCenterTest.java

```diff
@@ -16,11 +16,23 @@
 
 package de.saxsys.mvvmfx.utils.notifications;
 
+import de.saxsys.javafx.test.JfxRunner;
```

Comment:
We have moved the JfxRunner into mvvmFX and removed the old dependency. 
Therefore this import now has to be `de.saxsys.mvvmfx.testingutils.jfxrunner`